### PR TITLE
feature/news: add loader icon before updating news list

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: webapp/src/main/webapp/css/news.less [hunk 1]
```diff
@@ -2045,4 +2045,40 @@
       width: 70%;
     }
   }
-}
\ No newline at end of file
+}
+/** Loading Icon **/
+.lds-ring {
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/css/news.less]: I think we should use a more explicit name and use camelCase, for example : loadingRing.

File: webapp/src/main/webapp/news/components/NewsApp.vue [hunk 2]
```diff
@@ -33,6 +33,7 @@
         </div>
       </div>
     </div>
+    <div v-if="loadingNews" class="lds-ring"><div></div><div></div><div></div><div></div></div>
```
Comment (thomasdelhomenie) [inline webapp/src/main/webapp/news/components/NewsApp.vue]: You can use Vuetify component instead if you prefer : https://vuetifyjs.com/en/components/progress-circular

**Comment by samehaidi** [comment]
> I think that when a list of news is already displayed, the loading icon should be displayed somewhere in the filter bar instead of shifting the news :
> Screenshot from 2019-12-30 10-04-32
> @aelbenney @samehaidi WDYT ?

I agree

File: webapp/src/main/webapp/news/components/NewsApp.vue (hunk 2 above)
Comment (aelbenney) [reply webapp/src/main/webapp/news/components/NewsApp.vue]: okay, I will try it!

**Pull request closed without merging**